examples/visualization/plot_streamflow.jl

```diff
@@ -0,0 +1,61 @@
+# Visualization Code
+using ClimaRivers
+using JLD2, Plots
+
+data_file_path = joinpath(@__DIR__, "..", "..", "mini_data", "routing")
+simulation_result_file = joinpath(
+    data_file_path,
+    "simulations",
+    "simulations_lv05",
+    "gamma_IRF",
+    "streamflow_history120 days.jld2",
+)
```

Comment:
As this is directly related to the irf example, I think it would make sense to have this file in that directory